models/tax-category/src/tax-category-draft/presets/sample-data-fashion/standard-tax-category.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { TTaxCategoryDraft } from '../../../types';
+import standardTaxCategory from './standard-tax-category';
+
+describe('with a tax rate preset `standard tax category`', () => {
+  it('should return a tax category with name `Standard Tax Category` and 6 tax rates', () => {
```

Comment:
A GraphQL test as well since that's what we're going to be using?

Reply:
🤦

Reply:
@stephsprinkle Should I test for an empty `subRates` array?

Reply:
If it's empty, I don't think we need a test because we aren't using it.

Reply:
updated with [c1d1891](https://github.com/commercetools/test-data/pull/208/commits/c1d18917678ce44f8a348ff231e2c4c78548572c)